Makefile

```diff
@@ -1,61 +1,37 @@
-PKGS := $(shell go list ./... | grep -v go.uber.org/sally/vendor)
-SRCS := $(wildcard *.go)
-
 .PHONY: all
 all: test
 
-.PHONY: vendor-update
-vendor-update:
-	go get -v github.com/Masterminds/glide
-	glide update
-
-.PHONY: vendor-install
-vendor-install:
-	go get -v github.com/Masterminds/glide
-	glide install
-
 .PHONY: build
 build:
-	go build $(PKGS)
+	go build
 
 .PHONY: install
 install:
-	go install $(PKGS)
+	go install .
 
 .PHONY: lint
 lint:
-	go install ./vendor/github.com/golang/lint/golint
-	for file in $(SRCS); do \
-		golint $$file; \
-		if [ -n "$$(golint $$file)" ]; then \
-			exit 1; \
-		fi; \
-	done
+	go install github.com/golang/lint/golint
+	golint ./...
 
 .PHONY: vet
 vet:
-	go vet $(PKGS)
-
-.PHONY: errcheck
-errcheck:
-	go install ./vendor/github.com/kisielk/errcheck
-	errcheck $(PKGS)
+	go vet ./...
 
 .PHONY: staticcheck
 staticcheck:
-	go install ./vendor/honnef.co/go/tools/cmd/staticcheck
-	staticcheck $(PKGS)
+	go install honnef.co/go/tools/cmd/staticcheck
```

Comment:
`GO111MODULE` makes this module aware, right?

Reply:
Yep. With the env var set, it'll pick up the one we're pinned to.

Reply:
I'm more curious about `command -v staticcheck` in this case, would it also be somewhere scoped just to this path or global?

Reply:
No, the command is still installed globally. An option here is to do `go build -o $somepath $importPath` and that will leave the global version unchanged.

Reply:
Not a huge deal, I'm fine with either. I think second is slightly better practice but it's not any different than it was before

Reply:
Oh good call. I'll switch.